tmt/utils.py

```diff
@@ -1269,7 +1269,8 @@ def copytree(
     if not dirs_exist_ok or sys.version_info >= (3, 8):
         return cast(
             Path,
-            shutil.copytree(src=src, dst=dst, symlinks=symlinks, dirs_exist_ok=dirs_exist_ok))
+            shutil.copytree(src=src, dst=dst, symlinks=symlinks,
+                            dirs_exist_ok=dirs_exist_ok))  # type: ignore[call-arg]
```

Comment:
Please, add a small comment, similar to the one above `is_relative_to`, explaining why the `ignore` is needed. We'd like to keep them justified & documented.